dev/CommonStyles/TestUI/CommonStylesPage.xaml

```diff
@@ -7,6 +7,7 @@
     xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:AnimatedVisuals="using:Microsoft.UI.Xaml.Controls.AnimatedVisuals"
```

Comment:
nit: Settle for "AnimatedVisuals" or "animatedvisuals" or something else while we can so we don't fragment the naming scheme.